src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,33 @@
+@mixin hover($_property, $_toValue) {
+  transition: #{$_property} 0.3s;
+  &:hover {
+    #{$_property}: $_toValue;
+  }
+}
+
+@mixin onDesktop {
+  @media (min-width: 1024px) {
+    @content;
+  }
+}
+
+@mixin onTablet {
+  @media (min-width: 600px) {
+    @content;
+  }
+}
```

Comment:
600 is a bit too low for a tablet breakpoint

Reply:
- should not be pixel perfect tho
- made textarea resizable on purpose
- same with scroll
- I guess I'll pass on 600 width now